README.md

```diff
@@ -14,21 +11,14 @@
 
 ---
 
-WildBeast is a multifunction Discord bot intended to be able to perform various tasks, ranging from easy server moderation (Kick, ban etc.) to having fun (Cleverbot etc.) in your Discord server. Written in JavaScript [Node.JS](https://nodejs.org), using the [discordie](https://qeled.github.io/discordie) library.
+WildBeast is a multifunctional Discord bot, intended to provide a framework thats both easy to use and run, and easy to extend and modify.
```

Comment:
Any chance you could word this a little better? A lot of "and" stacking there.

Reply:
Changed locally to `WildBeast is a multifunctional Discord bot, intended to provide a framework that's easy to use, extend, and modify.`